pages/develop.md

```diff
@@ -235,7 +235,7 @@ Before you can build and install the extension, you need a developer certificate
 `git clone --recursive git@github.com:privly/privly-safari.git`
 2. Open Safari and choose "Show Extension Builder" from the Develop menu.
 3. Click the "+" button and choose "Add Extension...".
-4. Select the directory containing the safari extension (where you just cloned it to).
+4. Select the directory called "privly.safariextension" which is containing the safari extension and it is under the directory where you just cloned it to.
```

Comment:
i'd write this as:

Select the directory called "privly.safariextension" containing the safari extension inside the directory you just cloned.